File: client/ITI-graduation-project-Advilaw-Frontend-main/src/app/types/Jobs/ClientConsultationDTO.ts
export interface ClientConsultationDTO {
  id: number;
  header: string;
  description: string;
  budget: number;
  statusLabel: string;
  lawyerName: string;
  lawyerProfilePictureUrl: string;
  duration?: number | null;
  appointmentTime?: string | null;
} 

File: server/ITI-graduation-project-Advilaw-main/AdviLaw.Application/Features/JobSection/DTOs/ClientConsultationDTO.cs
using System;

namespace AdviLaw.Application.Features.JobSection.DTOs
{
    public class ClientConsultationDTO
    {
        public int Id { get; set; }
        public string Header { get; set; }
        public string Description { get; set; }
        public int Budget { get; set; }
        public string StatusLabel { get; set; }
        public string LawyerName { get; set; }
        public string LawyerProfilePictureUrl { get; set; }
        public int? Duration { get; set; }
        public DateTime? AppointmentTime { get; set; }
    }
} 